package.json

```diff
@@ -16,8 +16,7 @@
   "scripts": {
     "lint": "./node_modules/.bin/eslint ./src --ext .js,.jsx,.ts,.tsx",
     "build": "rollup -c",
-    "copy_publish_files": "cp ../README.md .",
-    "prepublish": "yarn run build && yarn run copy_publish_files"
+    "prepublish": "yarn run build"
```

Comment:
If we don't execute `cp ../README.md .` before building, we probably won't upload the README file to the npm package, causing the description to be blank there.

I think we should still keep this line.

Reply:
The beta version underwent a change to the directory structure. There is no longer a package subdirectory and the README file is still there it just lives at the top level so I don't see a need to copy the file